packages/babel-parser/test/fixtures/es2022/class-private-properties/invalid-object-method/output.json

```diff
@@ -2,7 +2,8 @@
   "type": "File",
   "start":0,"end":33,"loc":{"start":{"line":1,"column":0},"end":{"line":3,"column":1}},
   "errors": [
-    "SyntaxError: Private names can only be used as the name of a class element (i.e. class C { #p = 42; #m() {} } )\n or a property of member expression (i.e. this.#p). (2:11)"
+    "SyntaxError: Private names can only be used as the name of a class element (i.e. class C { #p = 42; #m() {} } )\n or a property of member expression (i.e. this.#p). (2:10)",
+    "SyntaxError: Private name #x is not defined. (2:10)"
```

Comment:
Can we throw the second error only when it would otherwise be a valid usage? The main problem here is not that `#x` is not defined, but that it's trying to use it to declare a private method in an object.